.github/workflows/update_operators.yml

```diff
@@ -126,7 +139,13 @@ jobs:
           title: Update generated code for DPF ${{ github.event.inputs.ANSYS_VERSION || '242' }}${{ github.event.inputs.standalone_branch_suffix || '' }} on ${{ github.ref_name }}
```

Comment:
What I'll eventually do is propagate from TFS the number of the last PR merged in DPF. I'll name the dpf-standalone PR and branch with this, and give it as input to this workflow to also name the PyDPF-Core PR and branch. That way instead of relying on something generic, we'll be able to know exactly which version of the server code we are synchronizing for.